### PR TITLE
mirror: add player name to title

### PR DESCRIPTION
**Pull request by MadCatzAno**
Changed the mirrored window title from "OpenOSRS" to "OpenOSRS Mirror" by default, because multiple windows with the same title caused bugs with OBS...

+ Added an option to appended player name to the title.

// Also, I fcked up my commit message first time, ignore it pls >_<

### CODE REVIEW

File: mirror/src/main/java/net/runelite/client/plugins/mirror/MirrorPlugin.java [hunk 1]
```diff
@@ -52,16 +56,32 @@
...
+	{
+		return configManager.getConfig(MirrorConfig.class);
+	}
+
 	public static JFrame jframe;
 	public static final Canvas canvas = new Canvas();
 	public static BufferedImage bufferedImage;
+	private static String title = "OpenOSRS Mirror";
+
+	private void updateTitle()
```
Comment (ThatGamerBlue) [inline mirror/src/main/java/net/runelite/client/plugins/mirror/MirrorPlugin.java:73]: make this update the title automatically so you dont need the class variable `title`

File: mirror/src/main/java/net/runelite/client/plugins/mirror/MirrorPlugin.java [hunk 2]
```diff
@@ -100,4 +120,19 @@ private void onDrawFinished(DrawFinished event)
...
+	private void onConfigChanged(ConfigChanged event)
+	{
+		if (!event.getGroup().equals("mirror"))
+		{
+			return;
+		}
+
+		if (jframe != null)
+		{
+			updateTitle();
```
Comment (ThatGamerBlue) [inline mirror/src/main/java/net/runelite/client/plugins/mirror/MirrorPlugin.java:134]: SwingUtilities.invokeLater

File: mirror/src/main/java/net/runelite/client/plugins/mirror/MirrorConfig.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+import net.runelite.client.config.Config;
+import net.runelite.client.config.ConfigGroup;
+import net.runelite.client.config.ConfigItem;
+
+@ConfigGroup("mirror")
+public interface MirrorConfig extends Config
+{
+	@ConfigItem(
+			position = 0,
```
Comment (ThatGamerBlue) [inline mirror/src/main/java/net/runelite/client/plugins/mirror/MirrorConfig.java:11]: these are indented one tab too far

**Comment by xKylee** [comment]
LGTM @ThatGamerBlue

**Pull request merged**